### ISSUE TITLE
Visualize arrow rotation

### ISSUE THREAD

**Issue by JoachimGoedhart** [DaetwylerStephan/draw_arrow_in_movies#3]
Thanks for creating this useful plugin!
It would be nice if the arrow rotation (set with the 'Arrow angle') can be shown in the 'Arrow preview' window.
(not sure how easy that would be)

**Comment by DaetwylerStephan** [comment]
I'm very glad to hear that the plugin is useful to you, and thank you for the tweet and comments!

In a future release, this would be indeed a nice addition to have. In the current version of the plugin, I decided against this for the following reasons: (i) I could only judge a good orientation of the arrows on the movie data itself (considering all timepoints). Therefore, I made adding and deleting of arrow overlays fast (especially when using keys on the keyboard). In this scheme, the 'Arrow preview' window serves for the design of the arrow, and the movie data for placement and orientation. (ii) I wanted to keep the input panel as small as possible for convenience. In its current size, rotating an arrow would mean that the 'Arrow preview' window would need to be rescaled in height, which would require rescaling of the input panel and might be inconvenient for fast clicking.
